lib/formats/map.js

```diff
@@ -23,6 +34,32 @@ export const mapEs = {
   },
 };
 
+
+/**
+ * JavaScript token map
+ * @type {import('style-dictionary').Format}
+ * @example ```js
+ * import { tokensMeta } from '@rhds/tokens/meta.js';
+ *
+ * const fontfamilyCssString = tokensMeta.get('--rh-font-family-body-text');
+ * // e.g. '{
+ * value: 'RedHatText, "Red Hat Text", sans-serif',
+ * description: 'The font family used for body text',
+ * extensions: ''
+ * }';
+ * ```
+ */
+export const metaMapEs = {
+  name: 'javascript/meta-map',
+  formatter({ file, dictionary }) {
+    return [
+      fileHeader({ file }),
+      `export const tokensMeta = new Map(${JSON.stringify(makeMetaEntries(dictionary), null, 2)});`,
```

Comment:
```suggestion
      `export const tokens = new Map(${JSON.stringify(makeMetaEntries(dictionary), null, 2)});`,
```

Reply:
Yeah that makes sense I was thinking it would conflict with the other `export const tokens` got overly protective without giving thought to how that is a seperate importable file anyways.

Reply:
```js
import { tokens as meta } from 'meta.js';
```